src/devcenter/azext_devcenter/aaz/latest/devcenter/admin/network_connection/__init__.py

```diff
@@ -12,9 +12,10 @@
 from ._create import *
 from ._delete import *
 from ._list import *
-from ._list_health_detail import *
+from ._list_health_check import *
```

Comment:
It seems the command name changed.

Reply:
Confirmed that the command name should be changed